jbpm-workitem-itests/src/test/resources/kjars/kafka-process/pom.xml

```diff
@@ -12,12 +12,14 @@
     <dependency>
       <groupId>org.apache.kafka</groupId>
       <artifactId>kafka-clients</artifactId>
-      <version>2.3.1</version>
+      <version>2.6.0</version>
+      <scope>provided</scope>
     </dependency>
     <dependency>
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-databind</artifactId>
-      <version>2.11.0</version>
+      <version>2.10.5.1</version>
```

Comment:
shouldn't we use/reuse the version property from kie-parent?

Reply:
@mareknovotny sure, good catch